internal/gps/vcs_source.go

```diff
@@ -353,6 +350,20 @@ type bzrSource struct {
 	baseVCSSource
 }
 
+func (s *bzrSource) exportRevisionTo(ctx context.Context, rev Revision, to string) error {
+	// TODO: use bzr instead of the generic approach in
```

Comment:
i have _very_ little faith that bzr would actually be faster 😉 to the point where it's not even worth having the TODO, i think